com.unity.netcode.gameobjects/package.json

```diff
@@ -7,8 +7,10 @@
     "dependencies": {
         "com.unity.modules.ai": "1.0.0",
```

Comment:
I still wonder why we depend on AI

Reply:
very good question. we used to have `NetworkNavMeshAgent` but we no longer have it. I'll double check and potentially remove AI package dependency.

Reply:
we don't, it's gone!

Reply:
Bravo - the world is now a slightly better place